test/lint/test_runner/src/main.rs

```diff
@@ -209,12 +214,18 @@ fn get_subtrees() -> Vec<&'static str> {
     ]
 }
 
-/// Return the pathspecs to exclude all subtrees
-fn get_pathspecs_exclude_subtrees() -> Vec<String> {
-    get_subtrees()
-        .iter()
-        .map(|s| format!(":(exclude){}", s))
-        .collect()
+/// Return the pathspecs to exclude by default
+fn get_pathspecs_default_excludes() -> Vec<String> {
+    {
+        let mut list = get_subtrees();
+        list.extend([
+            "doc/release-notes/release-notes-*", // archived notes
+        ]);
+        list
+    }
+    .iter()
```

Comment:
haven't tried it, but was wondering it this would also work with [chaining](https://doc.rust-lang.org/std/iter/struct.Chain.html#examples) the iterators?
```suggestion
    get_subtrees()
        .into_iter()
        .chain(["doc/release-notes/release-notes-*"]) // archived notes
```